Tests/Rules/UseConsistentWhitespace.tests.ps1

```diff
@@ -243,10 +255,132 @@ $x = @{
         It "Should not find a violation if a end of input follows a semi-colon" {
             $def = @'
 $x = "abc";
+'@
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings | Should -Be $null
+        }
+    }
+
+
+    Context "CheckPipe" {
+        BeforeAll {
+            $ruleConfiguration.CheckInnerBrace = $false
+            $ruleConfiguration.CheckOpenBrace = $false
+            $ruleConfiguration.CheckOpenParen = $false
+            $ruleConfiguration.CheckOperator = $false
+            $ruleConfiguration.CheckPipe = $true
+            $ruleConfiguration.CheckSeparator = $false
+        }
+
+        It "Should find a violation if there is no space after pipe" {
```

Comment:
Could these 2 be simplified into `Get-Item|foo` and look for 2 errors?

Reply:
No, because the existing test helper method `Test-CorrectionExtentFromContent` does not support more than 1 violation. We could adapt it but I generally prefer having more separated tests over tests that test 2 things.